ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -26,17 +26,30 @@ open class CommandBar: UIView {
 
     // MARK: - Public methods
 
+    @available(*, deprecated, renamed: "init(itemGroups:leadingItemGroups:trailingItemGroups:)")
```

Comment:
In retrospect, it's probably best to not mark this as deprecated quite yet. Maybe just mark it as "renamed" to allow for users to update and not break code.